components/record/WebResources.vue

```diff
@@ -3,7 +3,7 @@
     <b-list-group-item
       v-for="webResource in media"
       :key="webResource.rdfAbout"
-      :data-qa="'webResource ' + webResource.rdfAbout"
+      :data-qa="'web resource webResource ' + webResource.rdfAbout"
```

Comment:
A readme certainly makes sense. It wouldn't be component specific though, we can have QA elements at page level. Perhaps something more generic in our Wiki as opposed to in the repository?

I wouldn't be too concerned about the nestedSelector logic since I don't think we should specify anything as being "non xyz", as that would lead to describing everything as not anything but itself. As long as we only use "affirmatively descriptive" QA attributes we can use negative steps   like the ones with "I (can't|don't|shouldn't|won't) see..." to check for things not being/containing certain content.

Reply:
I agree with documenting the specifications for the data attribute. The wiki makes sense, the readme on the other hand is a bit closer to the actual code.